tests/sc_resume_partition.test/runit

```diff
@@ -8,35 +8,44 @@ set -ex
 [ -z "${CLUSTER}" ] && { echo "Test requires a cluster"; exit 0; }
 
 dbnm=$1
+tier="default"
 
-restart_cluster() {
-	set +e
-	for node in ${CLUSTER} ; do
-		kill_restart_node ${node} &
-	done
-	set -e
+downgrade_master() {
+	local master
+	master=$(get_master)
+	if (( $? != 0 )); then
+		echo "FAIL: Could not get master"
+		return 1
+	fi
 
-	sleep 2
+	if ! cdb2sql --host ${master} ${CDB2_OPTIONS} ${dbnm} ${tier} "exec procedure sys.cmd.send('downgrade')"; then
+		echo "FAIL: Could not send downgrade command to master"
+		return 1
+	fi
 
-	wait_for_cluster
+	local new_master=${master}
```

Comment:
there is a chance that the old master to still win the election, in which case this test will hang

Reply:
I'll change it to run downgrade in a loop until there's a new master, thanks!